account_invoice_line_sale_line_position/models/account_invoice.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo import api, fields, models
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    has_order_position = fields.Boolean(compute="_compute_has_order_position")
+
+    @api.depends("invoice_line_ids.position_formatted")
+    def _compute_has_order_position(self):
+        for record in self:
+            record.has_order_position = any(
```

Comment:
it would be better to loop on each line and stop at the 1st one having the value. Not a blocker tho,